plugins/plugin-chart-echarts/src/Gauge/controlPanel.tsx

```diff
@@ -0,0 +1,267 @@
+              label: t('Number format'),
+              renderTrigger: true,
+              default: numberFormat,
+              choices: D3_FORMAT_OPTIONS,
+              description: `${t('D3 format syntax: https://github.com/d3/d3-format')} ${t(
+                'Only applies when "Label Type" is set to show values.',
+              )}`,
+            },
+          },
+        ],
+        [
+          {
+            name: 'font_size',
+            config: {
+              type: 'SliderControl',
+              label: t('Font size'),
+              renderTrigger: true,
+              min: 10,
+              max: 25,
+              default: fontSize,
+              description: t('Font size for chart elements'),
+            },
+          },
+        ],
+        [<h1 className="section-header">{t('Progress Bar')}</h1>],
+        [
+          {
+            name: 'roundcap',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Roundcap'),
+              renderTrigger: true,
+              default: roundcap,
+              description: t('Style the ends of the progress bar with a roundcap'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'show_split_line',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Show split lines'),
+              renderTrigger: true,
+              default: showSplitLine,
+              description: t('Show split lines'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'show_axis_tick',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Show axis line ticks'),
+              renderTrigger: true,
+              default: showAxisTick,
+              description: t('Show axis line ticks'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'start_angle',
+            config: {
+              type: 'TextControl',
+              label: t('Start angle'),
+              renderTrigger: true,
+              default: startAngle,
+              description: t('Angle at which to start progress axis'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'end_angle',
+            config: {
+              type: 'TextControl',
+              label: t('End angle'),
+              renderTrigger: true,
+              default: endAngle,
+              description: t('Angle at which to end progress axis'),
+            },
+          },
+        ],
+        [<h1 className="section-header">{t('Intervals')}</h1>],
+        ['color_scheme'],
+        [
+          {
+            name: 'intervals',
+            config: {
+              type: 'TextControl',
+              label: t('Interval bounds'),
+              description: t(
+                'Comma-separated interval bounds, e.g. 2,4,5 for intervals 0-2, 2-4 and 4-5. Last number should match the value provided for MAX.',
```

Comment:
I tried adding values here,but chart partially disappeared.
![Screenshot_2021-03-10  DEV  Explore - covid_vaccines(1)](https://user-images.githubusercontent.com/12967587/110575860-a394bf00-8185-11eb-851d-746117927cb4.png)

Reply:
Passing interval bounds implies passing interval color indices as well. Have you added those? I can't be sure from the screenshot. Also 0 as the first one might not be needed.